guide/how-it-works/silent-payments.md

```diff
@@ -0,0 +1,360 @@
+      alt: .
+      caption: .
+    - file: contact-page
+      modalImage: clipboard-options-add-contacts-or-pay
+      alt: .
+      caption: .
+    - file: contacts-list
+      modalImage: clipboard-options-add-contacts-or-pay
+      alt: .
+      caption: .
+    - file: home-screen-with-contacts
+      modalImage: clipboard-options-add-contacts-or-pay
+      alt: .
+      caption: .
+
+images_sp-onboarding:
+    - file: setup-silent-payments-onboarding-i
+      modalImage: setup-silent-payments-onboarding-i
+      alt: .
+      caption: .
+    - file: setup-silent-payments-onboarding-ii
+      modalImage: setup-silent-payments-onboarding-ii
+      alt: .
+      caption: .
+    - file: setup-silent-payments-onboarding-iii
+      modalImage: setup-silent-payments-onboarding-iii
+      alt: .
+      caption: .
+
+images_send:
+    - file: send-review-sp
+      modalImage: send-review-sp
+      alt: .
+      caption: .
+    - file: send-review-with-contact
+      modalImage: send-review-with-contact
+      alt: .
+      caption: .
+    - file: send-address-error
+      modalImage: send-address-error
+      alt: .
+      caption: .
+
+images_test-txns:
+    - file: test-txn-send-review
+      modalImage: test-txn-send-review
+      alt: .
+      caption: .
+    - file: test-txn-send-home
+      modalImage: test-txn-send-home
+      alt: .
+      caption: .
+    - file: test-txn-confirm-finalise-txn
+      modalImage: test-txn-confirm-finalise-txn
+      alt: .
+      caption: .
+
+images_backup:
+    - file: backup-cloud-backup
+      modalImage: backup-cloud-backup
+      alt: .
+      caption: .
+    - file: backup-wallet-birthday
+      modalImage: backup-wallet-birthday
+      alt: .
+      caption: .
+
+images_recovery:
+    - file: recovery-wallet-birthday-prompt
+      modalImage: recovery-wallet-birthday-prompt
+      alt: .
+      caption: .
+    - file: recovery-wallet-birthday-enter
+      modalImage: recovery-wallet-birthday-enter
+      alt: .
+      caption: .
+---
+
+<!--
+
+Editor's notes
+
+This page provides best practices and designers for the first-use experience of the
+daily spending wallet reference design.
+
+Source designs for all screens can be found in this Figma file:
+https://www.figma.com/file/lf2Xyw2I2OXPsHiFQVQdiG/Daily-spending-wallet-prototype?node-id=4152%3A306968&t=boEAgdyqfi39SBoF-1
+
+Source file for header image:
+https://www.figma.com/file/NjtMNQiJtoVkedEHgwD0A9/BTC-DSN-Guide-Header-Images?node-id=0%3A1&t=JzVvsbPdFpjhNTGF-1
+
+-->
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/silent-payments/silent-payments-header.png"
+   retina = "/assets/images/guide/how-it-works/silent-payments/silent-payments-header@2x.png"
+   mobile = "/assets/images/guide/how-it-works/silent-payments/silent-payments-header-mobile.png"
+   mobileRetina = "/assets/images/guide/daily-spending-wallet/silent-payments/silent-payments-header-mobile@2x.png"
+   alt-text = "."
+   width = 1600
+   height = 1000
```

Comment:
Height is incorrect.

Reply:
Fixed in https://github.com/BitcoinDesign/Guide/pull/1109/commits/1617122d661dee455ee5d2e10d5e379472f540ca